app/views/hubstats/pull_requests/index.html.erb

```diff
@@ -10,8 +9,9 @@
         <input class="select2-control" id="users">
         </input>
       </div>
-
+      
```

Comment:
I find it funny that you are adding extra whitespace in some places, and actively removing extra whitespace elsewhere.